src/js/components/modals/AppModalComponent.jsx

```diff
@@ -98,7 +99,12 @@ var AppModalComponent = React.createClass({
     // All status below 300 are actually not an error
     if (status < 300) {
       this.onCreateApp();
-
+    } else if (status === 409) {
+      this.setState({
+        responseErrorMessages: {general: `Failed to change and deploy the new
```

Comment:
I was wondering if there was a file for messages! That's great, I will put it there. I like your message, but let's wait for @pierlo-upitup for confirmation.